pcsx2/gui/AppMain.cpp

```diff
@@ -620,7 +620,7 @@ void Pcsx2App::HandleEvent(wxEvtHandler* handler, wxEventFunction func, wxEvent&
 #ifndef DISABLE_RECORDING
 		if (g_Conf->EmuOptions.EnableRecordingTools)
 		{
-			if (g_RecordingControls.IsEmulationAndRecordingPaused())
+			if (g_InputRecordingControls.IsRecordingPaused())
```

Comment:
Naming this function is hard because it's not really tied to whether or not _recording_ is paused (you can frame-advance and pause outside of a recording of course).  But the old name was also inaccurate, how about:
`IsEmulationPausedByInputRecordingTools`

kinda verbose but, atleast it's accurate.

Reply:
We could probably get away with just IsPaused(). Adding ByInputRecordingTools is a bit redundant since it must be called by linking it with the `g_InputRecordingControls` object anyways.